api/liboni/oni-repl/Makefile

```diff
@@ -4,8 +4,17 @@ NAME      :=  oni-repl
 SRC       :=  main.c
 OBJ       :=  $(SRC:.c=.o)
 CFLAGS    :=  -pedantic -Wall -W -Werror -fPIC -O3 $(DEFS)
-LDFLAGS   :=  -lm -lpthread -loni -ldl
+# LDFLAGS   :=  -lm -lpthread -loni -ldl
```

Comment:
would prefer that a final decision is made since things like this can cause confusion in the future